frontend/src/pages/Orders.jsx

```
@@ -0,0 +1,8 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+
+const Orders = () => {
+  return <div></div>;
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)